.circleci/config.yml

```diff
@@ -224,12 +224,25 @@ jobs:
           name: Create Cloudflare DNS namespace
           command: node scripts/create-review-app-subdomain.js
 
+  check-npm-vulnerabilities:
+    executor: node-executor
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: yarn
+      - run:
+          name: Check for critical npm vulnerabilities
+          command: yarn npm audit --severity critical --recursive
```

Comment:
I assume this is enough, we regularly check Force deployments. But for some other services It might be nice to also send a message to Slack